insonmnia/locator/storage/libkv/storage_e2e_test.go

```diff
@@ -0,0 +1,131 @@
+package libkv_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/docker/libkv/store"
+	"github.com/docker/libkv/store/boltdb"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+
+	engine "github.com/sonm-io/core/insonmnia/locator/storage/libkv"
+
+	"encoding/json"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/docker/libkv"
+	ds "github.com/sonm-io/core/insonmnia/locator/datastruct"
+	"github.com/stretchr/testify/require"
+	"os"
```

Comment:
goimports?

Reply:
Go fmt. As far as I know, we've got a commit hook with go-fmt.

Reply:
goimports pass required checks